void f() {
    try {  }
    catch (std::bad_alloc) {  }
}
